### PR TITLE
bugfix #273: added delete popup to all admin pages

### PR DESCRIPTION
**Pull request by sifes**
- added delete popup in ui
- added missing delete popups in all admin pages

closes: #273 , closes: #274

### CODE REVIEW

File: fictadvisor-web/src/components/pages/admin/questions-admin-page/components/questions-search-page/components/questions-table/components/table-actions/TableActions.styles.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { SxProps, Theme } from '@mui/material/styles';
+
+export const tableColumn: SxProps<Theme> = {
+  flexDirection: 'row',
+  gap: '8px',
+  alignItems: 'center',
+};
```
Comment (Markusplay) [inline fictadvisor-web/src/components/pages/admin/questions-admin-page/components/questions-search-page/components/questions-table/components/table-actions/TableActions.styles.ts:7]: forgot `display: flex`

File: fictadvisor-web/src/components/pages/admin/disciplines-admin-page/components/disciplines-table/components/table-actions/TableActions.styles.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { SxProps, Theme } from '@mui/material/styles';
+
+export const tableColumn: SxProps<Theme> = {
+  flexDirection: 'row',
```
Comment (Markusplay) [inline fictadvisor-web/src/components/pages/admin/disciplines-admin-page/components/disciplines-table/components/table-actions/TableActions.styles.ts:4]: same here

File: fictadvisor-web/src/components/pages/admin/admin-teachers/edit-teachers-admin-page/components/header-edit/HeaderEdit.tsx [hunk 3]
```diff
@@ -12,6 +12,8 @@ import {
 } from '@/components/common/ui/button-mui/types';
 import { Teacher } from '@/types/teacher';
 
+import DeletePopup from '../../../../../../common/ui/delete-popup/DeletePopup';
```
Comment (Markusplay) [inline fictadvisor-web/src/components/pages/admin/admin-teachers/edit-teachers-admin-page/components/header-edit/HeaderEdit.tsx:15]: use alias for imports

**Review by Markusplay — COMMENTED** [review]
DRY is broken with styles:
`
import { SxProps, Theme } from '@mui/material/styles';

export const tableColumn: SxProps<Theme> = {
  flexDirection: 'row',
  gap: '8px',
  alignItems: 'center',
};

export const button: SxProps<Theme> = {
  borderRadius: '8px',
  width: '157px',
};
`

File: fictadvisor-web/src/components/pages/admin/disciplines-admin-page/components/disciplines-table/components/table-actions/TableActions.styles.ts (hunk 2 above)
Comment (sifes) [reply fictadvisor-web/src/components/pages/admin/disciplines-admin-page/components/disciplines-table/components/table-actions/TableActions.styles.ts:4]: we don`t need a display: flex here, as there is a Stack from mui, which have display: flex by default

File: fictadvisor-web/src/components/pages/admin/questions-admin-page/components/questions-search-page/components/questions-table/components/table-actions/TableActions.styles.ts (hunk 1 above)
Comment (sifes) [reply fictadvisor-web/src/components/pages/admin/questions-admin-page/components/questions-search-page/components/questions-table/components/table-actions/TableActions.styles.ts:7]: we don`t need a display: flex here, as there is a Stack from mui, which have display: flex by default

**Review by Markusplay — APPROVED** [review]

**Review by hoshion — APPROVED** [review]

**Pull request merged**